<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>524</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>云对讲信息发布测试工具</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>40</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>本机Ip地址：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_ip">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>40</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_port">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>100</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>19180</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_password">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>130</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>hori2015</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>160</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>通讯协议版本号：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_ver">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>160</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>1.0</string>
    </property>
    <property name="maxLength">
     <number>5</number>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>10</y>
      <width>231</width>
      <height>251</height>
     </rect>
    </property>
    <property name="title">
     <string>基础信息</string>
    </property>
    <widget class="QComboBox" name="box_cmd">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>210</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>GoHome</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>LeaveHome</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>请求消息ID：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_deviceid">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>180</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>+868320000140000000000000C</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>通讯密码：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_port_native">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>60</y>
       <width>50</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>19179</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>54</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>本机端口：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>第三方设备端口：</string>
     </property>
    </widget>
   </widget>
   <widget class="QTextBrowser" name="txt_rx">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>321</width>
      <height>431</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>290</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>业主当前所在楼层号：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_basefloor">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>290</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>320</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>业主家楼栋单元号楼层号房号：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_home">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>320</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>340</y>
      <width>381</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>楼栋单元号、楼层与房号请用&quot;逗号&quot;分隔,同时根据协议,楼栋单元号可被省略,</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>380</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>触发事件类型：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="box_triggertype">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>380</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>1-&gt;用户刷住户卡</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2-&gt;用户使用住户密码</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3-&gt;访客使用访客密码</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4-&gt;住户使用app远程操作</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5-&gt;按出门按键</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6-&gt;保安使用保安密码</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>7-&gt;保安刷保安卡</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>410</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>卡号：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_cardnum">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>410</y>
      <width>311</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>440</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>开启门锁：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="box_unlock">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>440</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>1-&gt;开启</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>0-&gt;不开启</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>440</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>门锁编号：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="box_locknum">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>440</y>
      <width>31</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>440</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>读卡器编号：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="box_readernum">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>440</y>
      <width>31</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>7</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>8</string>
     </property>
    </item>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>270</y>
      <width>391</width>
      <height>231</height>
     </rect>
    </property>
    <property name="title">
     <string>回家请求信息</string>
    </property>
    <widget class="QPushButton" name="button_gohome_request">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>200</y>
       <width>121</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>发送回家请求信息</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>30</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>业主家的楼层房号：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_home_l">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>50</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>80</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>业主去往楼层：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_toroom_l">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>100</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>130</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>触发事件类型：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_leavehome_request">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>220</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>发送离家请求信息</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>10</y>
      <width>151</width>
      <height>251</height>
     </rect>
    </property>
    <property name="title">
     <string>离家请求信息</string>
    </property>
    <widget class="QComboBox" name="box_triggertype_l">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>101-&gt;用户进行招梯</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>102-&gt;用户进行离家布防</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QPushButton" name="button_refresh">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>470</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>清空来往信息</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>341</width>
      <height>491</height>
     </rect>
    </property>
    <property name="title">
     <string>来往信息监控</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>190</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>位置编码：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>360</y>
      <width>361</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>多组套间请用&quot;分号&quot;作为分割符。</string>
    </property>
   </widget>
   <zorder>groupBox_2</zorder>
   <zorder>groupBox_3</zorder>
   <zorder>groupBox</zorder>
   <zorder>groupBox_4</zorder>
   <zorder>label</zorder>
   <zorder>txt_ip</zorder>
   <zorder>txt_port</zorder>
   <zorder>txt_password</zorder>
   <zorder>label_5</zorder>
   <zorder>txt_ver</zorder>
   <zorder>txt_rx</zorder>
   <zorder>label_7</zorder>
   <zorder>txt_basefloor</zorder>
   <zorder>label_8</zorder>
   <zorder>txt_home</zorder>
   <zorder>label_9</zorder>
   <zorder>label_10</zorder>
   <zorder>box_triggertype</zorder>
   <zorder>label_11</zorder>
   <zorder>txt_cardnum</zorder>
   <zorder>label_12</zorder>
   <zorder>box_unlock</zorder>
   <zorder>label_13</zorder>
   <zorder>box_locknum</zorder>
   <zorder>label_14</zorder>
   <zorder>box_readernum</zorder>
   <zorder>label_15</zorder>
   <zorder>txt_home_l</zorder>
   <zorder>label_16</zorder>
   <zorder>txt_toroom_l</zorder>
   <zorder>label_17</zorder>
   <zorder>button_leavehome_request</zorder>
   <zorder>button_refresh</zorder>
   <zorder>label_3</zorder>
   <zorder>label_19</zorder>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
